packages/kit/src/runtime/client/client.js

```diff
@@ -1513,6 +1528,54 @@ export function create_client({ target, base }) {
 				});
 			});
 
+			target.addEventListener('submit', (event) => {
+				if (event.defaultPrevented) return;
+
+				const form = /** @type {HTMLFormElement} */ (
+					HTMLFormElement.prototype.cloneNode.call(event.target)
+				);
+
+				const submitter = /** @type {HTMLButtonElement | HTMLInputElement} */ (event.submitter);
```

Comment:
```suggestion
				const submitter = /** @type {HTMLButtonElement | HTMLInputElement | null} */ (event.submitter);
```

danger, will robinson

Reply:
We know this always be defined